game.js

```diff
@@ -0,0 +1,62 @@
+function drawmap(){
```

Comment:
drawmap() { - use space between round bracket and '{' symbol. 
What IDE do you use?